frontend/lib/src/components/widgets/DownloadButton/DownloadButton.tsx

```diff
@@ -50,7 +54,11 @@ function DownloadButton(props: Props): ReactElement {
     const link = document.createElement("a")
     const uri = endpoints.buildMediaURL(element.url)
     link.setAttribute("href", uri)
-    link.setAttribute("target", "_self")
+    if (enforceDownloadInNewTab) {
+      link.setAttribute("target", "_blank")
+    } else {
+      link.setAttribute("target", "_self")
+    }
```

Comment:
Would be good to add a unit test for this if possible (I'm assuming we can check link attributes using RTL?)

Reply:
At first, I tried to write e2e tests, but it turned out to be a little problematic; the playwright does not distinguish downloading a file in a new tab.


checking the attributes themselves is a bit problematic because we are not mounting a link to the DOM.

I separated link creation into a function and added js unit tests for that!

Thanks for review!